src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt

```diff
@@ -83,6 +83,7 @@ fun createSpacingBuilder(commonSettings: CommonCodeStyleSettings, rustSettings:
         .between(IDENTIFIER, GENERIC_ARGS).spaceIf(false)
         .between(IDENTIFIER, ARG_LIST).spaceIf(false)
         .between(GENERIC_PARAMS, PARAMS_LIKE).spaceIf(false)
+        .between(GENERIC_ARGS, ARG_LIST).spaceIf(false)
         .beforeInside(ARG_LIST, CALL_EXPR).spaceIf(false)
```

Comment:
Unfortunately, no. These guys from test have totally different PSI.

`SpacingBuilder`'s rules apply only to neighbouring siblings in the AST. That's why this didn't work for method call's.
![idea_2016-07-14_21-01-02](https://cloud.githubusercontent.com/assets/3450050/16851989/54c8d0a2-4a06-11e6-8371-3fd7cd1b331e.png)